lib/api/core/models/repositories/userRepository.js

```diff
@@ -81,6 +107,31 @@ module.exports = function (kuzzle) {
     return deferred.promise;
   };
 
+  UserRepository.prototype.loadByUsernameAndPassword = function (username, password) {
```

Comment:
I'm not sure to get it. So when we want to install new auth plugin with another strategy, we have to add a function here for this specific authentication ?